src/plugins/dashboard/public/application/lib/save_dashboard.ts

```diff
@@ -42,24 +43,18 @@ import { DashboardStateManager } from '../dashboard_state_manager';
  * dashboard.
  */
 export function saveDashboard(
-  toJson: (obj: any) => string,
```

Comment:
if the case please remove the comment in line 38.

Reply:
Not sure whether there was a reason not to use `JSON.stringify` but it has been working fine so far. Will remove the comment in line 38.